modules/nixpkgs.nix

```diff
@@ -54,7 +54,7 @@ let
     merge = lib.mergeOneOption;
   };
 
-  _pkgs = import <nixpkgs> (
+  _pkgs = import pkgsPath (
```

Comment:
As mentioned in another PR of yours, this module should not be used when running it as flake IMO. Seems to be a bit impure to me. But I'm open for a discussion.